src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -542,7 +542,7 @@ def _validate_vm_vmss_create_vnet(cmd, namespace, for_scale_set=False):
     nics = getattr(namespace, 'nics', None)
 
     if vnet and '/' in vnet:
-        raise CLIError("incorrect '--vnet-name' usage: '--vnet-name' must be a name, not an ID.")
+        raise CLIError("incorrect '--vnet-name' usage: --subnet SUBNET_ID | --subnet SUBNET_NAME --vnet-name VNET_NAME")
```

Comment:
`raise CLIError('incorrect usage: ( --subnet ID | --subnet NAME --vnet-name NAME)')`

Reply:
@tjprescott do things look good now?

Reply:
No. The other usage errors simply state NAME or ID, not SUBNET_NAME or SUBNET_ID.

Reply:
I was trying to be consistent with the other usage error in the validator. Are we looking at the same changes [here](https://github.com/Azure/azure-cli/pull/7824/files)?

Reply:
Yes, they should all just same NAME and ID